src/main/pages/che-7/end-user-guide/ref_devfile-reference.adoc

```diff
@@ -440,7 +440,7 @@ A devfiles may only need the `CHE_PROJECTS_ROOT` environment variable to locate
 
 ==== Endpoints
 
-You can specify the endpoints that the docker image exposes. These endpoints can be made accessible to the users if the {prod-short} cluster is running using a Kubernetes ingress or an OpenShift route and to the other components within the workspace. You can create an endpoint for your application or database, if your application or database server is listening on a port and you want to be able to directly interact with it yourself or you want other components to interact with it.
+You can specify the endpoints that the Docker image exposes. These endpoints can be made accessible to the users if the {prod-short} cluster is running using a Kubernetes ingress or an OpenShift route and to the other components within the workspace. You can create an endpoint for your application or database, if your application or database server is listening on a port and you want to be able to directly interact with it yourself or you want other components to interact with it.
```

Comment:
```suggestion
You can specify the endpoints that the container image exposes. These endpoints can be made accessible to the users if the {prod-short} cluster is running using a Kubernetes ingress or an OpenShift route and to the other components within the workspace. You can create an endpoint for your application or database, if your application or database server is listening on a port and you want to be able to directly interact with it yourself or you want other components to interact with it.
```

Reply:
Please file an issue for it. That should be fixed as a separate PR throughout all of the files in both branches, since this is not the only instance of `Docker` we have.